deploy/csi-azurelustre-controller.yaml

```diff
@@ -71,10 +71,10 @@ spec:
             - "-v=5"
             - "--endpoint=$(CSI_ENDPOINT)"
           ports:
-            - containerPort: 29632
+            - containerPort: 29762
               name: healthz
               protocol: TCP
-            - containerPort: 29634
+            - containerPort: 29764
```

Comment:
The metrics port can be deleted. The metrics handler has been removed before in thie [PR 37](https://github.com/kubernetes-sigs/azurelustre-csi-driver/pull/37).

Reply:
Sure, I comment it out. Leave the port in deployment yaml so we won't forget the port num in the future